scripts/quests/otherAreas/Go_Go_Gobmuffin.lua

```diff
@@ -0,0 +1,138 @@
+}
+
+local nmStatus = function(player, mob)
+    local nmKilled = player:getLocalVar('NMKilled')
+    local nmBitVal = mob:getLocalVar('Req')
+
+    nmKilled = bit.bor(nmKilled, nmBitVal)
+    if bit.band(nmKilled) == 7 then -- Player Var has all 3 Bits (0b111)
+        quest:setVar(player, 'Prog', 1)
+    else
+        player:setLocalVar('NMKilled', nmKilled)  -- Resets upon zoning causing the player to need to refight the NMs
+    end
+end
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE and
+                player:hasCompletedMission(xi.mission.log_id.COP, xi.mission.id.cop.SHELTERING_DOUBT)
+        end,
+
+        [xi.zone.TAVNAZIAN_SAFEHOLD] =
+        {
+            ['Epinolle'] = quest:progressEvent(232),
+
+            onEventFinish =
+            {
+                [232] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_ACCEPTED
+        end,
+
+        [xi.zone.TAVNAZIAN_SAFEHOLD] =
+        {
+            ['Epinolle'] =
+            {
+                onTrigger = function(player, npc)
+                    if quest:getVar(player, 'Prog') == 1 then
+                        return quest:progressEvent(234)
+                    else
+                        return quest:event(233)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [234] = function(player, csid, option, npc)
+                    quest:complete(player)
+                end,
+            },
+        },
+
+        [xi.zone.RIVERNE_SITE_B01] =
+        {
+            ['Book_Browser_Bokabraq'] =
+            {
+                onMobDeath = function(mob, player, optParams)
+                    if quest:getVar(player, 'Prog') == 0 then
+                        nmStatus(player, mob)
+                    end
+                end,
+            },
+
+            ['Chemical_Cook_Chemachiq'] =
+            {
+                onMobDeath = function(mob, player, optParams)
+                    if quest:getVar(player, 'Prog') == 0 then
+                        nmStatus(player, mob)
+                    end
+                end,
+            },
+
+            ['Spell_Spitter_Spilospok'] =
+            {
+                onMobDeath = function(mob, player, optParams)
+                    if quest:getVar(player, 'Prog') == 0 then
+                        nmStatus(player, mob)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [17] = function(player, csid, option, npc)
+                    if
+                        quest:getVar(player, 'Prog') == 0 and
+                        not GetMobByID(riverneID.mob.SPELL_SPITTER_SPILUSPOK):isSpawned() and
```

Comment:
Each of these calls to GetMobByID could return nil, so chaining them immediately with `:isSpawned()` isn't safe. LLS is going to (or should) complain about this